app/uk/gov/hmrc/testuser/models/TestUser.scala

```diff
@@ -35,12 +35,13 @@ object ServiceKeys extends Enumeration {
   val AGENT_SERVICES: ServiceKeys.Value             = Value("agent-services")
   val LISA: ServiceKeys.Value                       = Value("lisa")
   val SECURE_ELECTRONIC_TRANSFER: ServiceKeys.Value = Value("secure-electronic-transfer")
-  val RELIEF_AT_SOURCE: ServiceKeys.Value           = Value("relief-at-source")
-  val CUSTOMS_SERVICES: ServiceKeys.Value           = Value("customs-services")
-  val GOODS_VEHICLE_MOVEMENTS: ServiceKeys.Value    = Value("goods-vehicle-movements")
-  val SAFETY_AND_SECURITY: ServiceKeys.Value        = Value("safety-and-security")
-  val CTC: ServiceKeys.Value                        = Value("common-transit-convention-traders")
-  val CTC_LEGACY: ServiceKeys.Value                 = Value("common-transit-convention-traders-legacy")
+  val RELIEF_AT_SOURCE: ServiceKeys.Value = Value("relief-at-source")
+  val CUSTOMS_SERVICES: ServiceKeys.Value = Value("customs-services")
+  val GOODS_VEHICLE_MOVEMENTS: ServiceKeys.Value = Value("goods-vehicle-movements")
+  val IMPORT_CONTROL_PRESENTATION_OF_GOODS: ServiceKeys.Value = Value("import-control-presentation-of-goods")
+  val SAFETY_AND_SECURITY: ServiceKeys.Value = Value("safety-and-security")
+  val CTC: ServiceKeys.Value = Value("common-transit-convention-traders")
+  val CTC_LEGACY: ServiceKeys.Value = Value("common-transit-convention-traders-legacy")
 }
```

Comment:
Can you please keep the formatting consistent with the previous implementation.

Reply:
As per the comments, formatting changes are done